controllers/deactivation/deactivation_controller.go

```diff
@@ -128,6 +142,12 @@ func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.
 		if err := r.resetDeactivatingState(ctx, usersignup); err != nil {
 			return reconcile.Result{}, err
 		}
+		// Set the scheduled deactivation time to nil
+		if err := statusUpdater.SetScheduledDeactivationStatus(ctx, usersignup, nil); err != nil {
```

Comment:
what is the expected UX for:
1. user won't be deactivated (is either excluded or is assigned to non-deactivating tier)
2. the value cannot be not calculated for the account for some reasons

Do you want to show the same message/value for these two cases?

Reply:
I guess since we can't actually differentiate between these two cases currently then a generic message or indicator would be most appropriate, perhaps for the "days remaining" box/field in the dashboard it can just display a `-` symbol or something.

Reply:
Yeah. Or `N/A`. Which can mean both `Not Available` and `Not Applicable` :)